packages/recommend/src/types/FrequentlyBoughtTogetherQuery.ts

```diff
@@ -2,5 +2,6 @@ import { RecommendationsQuery } from './RecommendationsQuery';
 
 export type FrequentlyBoughtTogetherQuery = Omit<
   RecommendationsQuery,
-  'model' | 'fallbackParameters'
->;
+  'model' | 'fallbackParameters' | 'facetName' | 'facetValue'
+> &
+  Required<Pick<RecommendationsQuery, 'objectID'>>;
```

Comment:
I think utility types makes it less readable, I'd personally be fine with having both `RecommendationsQuery` and `TrendingsQuery` defined on their own in that case